_includes/configuration/style-formats.md

```diff
@@ -50,4 +50,12 @@ style_formats: [
 ]
 ```
 
-The [Custom formats]({{ site.baseurl }}/demo/format-custom/) demo contains examples of custom style formats, including how to specify new formats and group headings.
+## Live Examples
+
+This example shows you how to override the built in [formats]({{ site.baseurl }}/configure/content-formatting/#formats) and add some custom styles to the `styleselect` dropdown toolbar button and the `formats` menu item using the [style_formats]({{ site.baseurl }}/configure/editor-appearance/#style_formats) configuration option.
```

Comment:
This sentence is very long. Should we shorten it/break it up?

Reply:
Maybe fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb